components/butterfly/tests/integration.rs

```diff
@@ -86,14 +86,8 @@ fn six_members_unmeshed_partition_and_rejoin_no_persistent_peers() {
 #[test]
 fn six_members_unmeshed_partition_and_rejoin_persistent_peers() {
     let mut net = btest::SwimNet::new(6);
-    net[0].member
-          .write()
-          .expect("Member lock is poisoned")
-          .set_persistent();
-    net[4].member
-          .write()
-          .expect("Member lock is poisoned")
-          .set_persistent();
+    net[0].member.write().set_persistent();
+    net[4].member.write().set_persistent();
```

Comment:
One side effect of this work I really appreciate is the elimination of a profusion of `expect`s.